INSTALL.md

```diff
@@ -106,10 +128,10 @@ then create a virtual environment (on Ubuntu 16.04 we needed to `export LC_ALL=C
 
 and follow the [basic instructions](#basics).
 
-To compile custom ops first install the additional system tools
+To compile custom ops first install docker
 
 ```
-$ sudo apt install -y libtool automake g++ curl
+./ $ apt install docker.io
```

Comment:
could it be `sudo apt install -y docker.io` to fit with the others?

Reply:
Yep